src/main/java/net/pms/util/StringUtil.java

```diff
@@ -41,6 +41,18 @@
 	public static final String SEC_TIME_FORMAT = "%02d:%02d:%02.0f";
 	public static final String DURATION_TIME_FORMAT = "%02d:%02d:%05.2f";
 	public static final String NEWLINE_CHARACTER = System.getProperty("line.separator");
+	public static final long KIBI = 1L << 10;
```

Comment:
@Sami32 We've had these units around since 1995, and standardized since 1999. They've been [adoption by IEC, NIST and ISO](https://en.wikipedia.org/wiki/Binary_prefix#Adoption_by_IEC.2C_NIST_and_ISO). The use of the SI units for both 1000- and 1024-based units has caused enough confusion in the world IMO. It's understandable than the SI units were back in the days what the correct units didn't exist. But to cling to them today, soon 20 years later, makes no sense.

The method handles both notations, but it would be very confusing if we suddenly were to specify RAM in SI units, we would get "too high" values compared to other references and it would cause even more confusion.